backend/services/downloadService.js

```diff
@@ -0,0 +1,10 @@
+const allowDownload = (response) => {
+  return (
+    response.locals?.user?.maintainAccessAdmin ||
+    (response.locals?.user?.maintainAuthUsers && !response.locals?.user?.groupManager)
```

Comment:
don't need the group manager check here - see https://github.com/ministryofjustice/manage-hmpps-auth-accounts/blob/main/backend/middleware/currentUser.js#L18
tbh would be better to create an `isAdmin` in `currentUser` and then can just check that rather having to muck about here - I might give that improvement a go afterwards...

Reply:
I'm confused then! I will leave it in your capable hands :smile: